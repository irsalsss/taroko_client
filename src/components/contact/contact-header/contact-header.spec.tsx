import { render } from "@/utils/test/react-testing-setup";
import { wrapperReactQuery } from "@/utils/test/wrapper-testing";
import ContactHeader from "./contact-header";

describe("ContactHeader", () => {
  it("should render ContactHeader successfully", () => {
    const { baseElement } = render(<ContactHeader onOpenModalAdd={jest.fn} />, {
      wrapper: wrapperReactQuery,
    });

    expect(baseElement).toMatchSnapshot();
    expect(baseElement).toBeTruthy();
  });
});
