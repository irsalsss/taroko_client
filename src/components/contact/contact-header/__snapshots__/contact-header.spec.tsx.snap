// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactHeader should render ContactHeader successfully 1`] = `
<body>
  <div>
    <div
      class="flex flex-wrap items-center justify-between w-full px-4 gap-4"
    >
      <h3>
        Contact List
      </h3>
      <div
        class="relative flex gap-1 flex-col"
      >
        <div
          class="input-icon"
        >
          <svg
            fill="none"
            height="15"
            viewBox="0 0 15 15"
            width="15"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M10 6.5C10 8.433 8.433 10 6.5 10C4.567 10 3 8.433 3 6.5C3 4.567 4.567 3 6.5 3C8.433 3 10 4.567 10 6.5ZM9.30884 10.0159C8.53901 10.6318 7.56251 11 6.5 11C4.01472 11 2 8.98528 2 6.5C2 4.01472 4.01472 2 6.5 2C8.98528 2 11 4.01472 11 6.5C11 7.56251 10.6318 8.53901 10.0159 9.30884L12.8536 12.1464C13.0488 12.3417 13.0488 12.6583 12.8536 12.8536C12.6583 13.0488 12.3417 13.0488 12.1464 12.8536L9.30884 10.0159Z"
              fill="currentColor"
              fill-rule="evenodd"
            />
          </svg>
        </div>
        <input
          class="input input-padding-with-icon"
          placeholder="search anything here..."
          type="text"
          value=""
        />
      </div>
      <button
        class="button"
        label="Add"
        role="button"
      >
        Add
      </button>
    </div>
  </div>
</body>
`;
